From 03204fe315e687ce75877101c5630f8c4e48da7e Mon Sep 17 00:00:00 2001
From: Cedric Bondier <cedricx.bondier@intel.com>
Date: Mon, 7 Jan 2013 15:59:34 +0100
Subject: [PORT FROM MAIN][BT] Multidevice support: BCM vendor specific library configuration

BZ: 76413

Configuring Broadcom vendor specific library to be built along with
bluedroid, necessary to support Broadcom BT chipset

Change-Id: I3deca6b38bab017601f54012fa84266d0208248e
Orig-Change-Id: I9d4d3787c52fac2ed5adcb7f175b92c97e9b2423
Feature: BT
Category: Integration
Purpose: ABSP-Adapter
Origin: Internal
Upstream-candidate: no
Signed-off-by: Raffaele Aquilone <raquiloX>
Reviewed-on: http://android.intel.com:8080/85370
Reviewed-by: Bondier, CedricX <cedricx.bondier@intel.com>
Reviewed-by: Aquilone, RaffaeleX <raffaelex.aquilone@intel.com>
Reviewed-by: Biselli, MirkoX <mirkox.biselli@intel.com>
Reviewed-by: Chen, HuaqiangX <huaqiangx.chen@intel.com>
Reviewed-by: Yao, Yong <yong.yao@intel.com>
Tested-by: Champciaux, Nicolas <nicolas.champciaux@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 libbt/Android.mk |    4 ----
 1 files changed, 0 insertions(+), 4 deletions(-)

diff --git a/libbt/Android.mk b/libbt/Android.mk
index 4b5ce4d..3ce6fc7 100644
--- a/libbt/Android.mk
+++ b/libbt/Android.mk
@@ -1,7 +1,5 @@
 LOCAL_PATH := $(call my-dir)
 
-ifneq ($(BOARD_HAVE_BLUETOOTH_BCM),)
-
 include $(CLEAR_VARS)
 
 BDROID_DIR := $(TOP_DIR)external/bluetooth/bluedroid
@@ -42,5 +40,3 @@ endif
 ifeq ($(TARGET_PRODUCT), full_wingray)
     include $(LOCAL_PATH)/conf/moto/wingray/Android.mk
 endif
-
-endif # BOARD_HAVE_BLUETOOTH_BCM
-- 
1.7.4.1

